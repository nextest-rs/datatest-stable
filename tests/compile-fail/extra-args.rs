datatest_stable::harness! {
    { test = my_test, root = "abc", pattern = ".*", extra = "xyz" }
}
