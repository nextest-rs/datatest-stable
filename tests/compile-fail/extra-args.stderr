error: unexpected extra arguments: extra = "xyz",
 --> tests/compile-fail/extra-args.rs:1:1
  |
1 | / datatest_stable::harness! {
2 | |     { test = my_test, root = "abc", pattern = ".*", extra = "xyz" }
3 | | }
  | |_^
  |
  = note: this error originates in the macro `$crate::harness_collect` which comes from the expansion of the macro `datatest_stable::harness` (in Nightly builds, run with -Z macro-backtrace for more info)
